build.sbt

```diff
@@ -13,7 +13,7 @@ lazy val scalaTestVersion  = "3.1.1"
 lazy val scalacheckVersion = "1.14.3"
 
 // Library versions
-lazy val reactiveMongoVersion = "1.0.0-noshaded-rc.2"
```

Comment:
Out of curiosity; this switches from `noshaded`; is there a reason here for that? Not sure what's recommended in this case and wanted to check + learn.

Reply:
Noshaded is only for advanced user who want to provide netty by themselves. Not recommended for dev who are not sure and have hard time to adjust version requirements.